tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>Where is the function <code>call</code>   used and what does it do? It's not really clear to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>It is for users to write the forward pass, which will be called in either the built-in training function fit() or evaluate() or custom training loop (e.g. tf.GradietTape context manager).</p>

Reply:
<p>This section is deleted in the latest version according to what's required.</p>